packages/core/src/logging/logger.ts

```diff
@@ -108,10 +117,10 @@ export class Logger {
 	}
 
 	private scopeFilter() {
-		return winston.format((info: TransformableInfo & { metadata: LogMetadata }) => {
+		return winston.format((info: TransformableInfo) => {
 			if (!this.isScopingEnabled) return info;
 
-			const { scopes } = info.metadata;
+			const { scopes } = (info as unknown as { metadata: LogMetadata }).metadata;
```

Comment:
Why is this better than the original?

Reply:
not sure why, but I'm getting this error 
![image](https://github.com/user-attachments/assets/848e9842-e59b-4539-96df-a923eb6cfaa1)

Reply:
So confused, this should've been erroring out in `cli` then? Also the `TransformableInfo` type is left open `[key: string | symbol]: any;` so worst case we shouldn't need the `unknown` intermediary or? `info.metadata as LogMetadata`